doc/source/NewSiteConfigs.rst

```diff
@@ -138,6 +138,7 @@ These instructions are meant to be a reference that users can follow to set up t
    brew install curl
    brew install cmake
    brew install openssl
+   brew install pkg-config
```

Comment:
On my mac I've built spack-stack-1.4.0 and the latest develop branches successfully and in both cases spack-stack built pkg-config, libevent and openssl. Does this work for your build of spack-stack-1.4.1? If so, we can keep the homebrew requirements to a minimum.

Reply:
This did not work using spack-stack 1.4.1.

Reply:
Update to this test. I also tried to use 1.4.0 with a brand new Homebrew installation and received the same error. I have attached the output of `spack concretize` for 1.4.0. 

[spack_con_output.txt](https://github.com/JCSDA/spack-stack/files/12176841/spack_con_output.txt)

Reply:
The concretize log is indicating that openssl is a dependency for libevent, and neither of those are dependencies for pkg-config. The error you reported occurred during the build of libevent and was complaining about not being able to find openssl.

Comparing your concretize log to my 1.4.0 build indicates these two differences:
- Your mac is intel based, while mine is arm64 based
- You are using openssl@3.1.1, while I am using openssl@1.1.1t

It doesn't make sense to me that installing pkg-config in homebrew makes the libevent problem go away since pkg-config and libevent are unrelated. I wonder if the version of openssl got changed via a chain of dependencies initiated by the pkg-config install.

Do you have the concretize log from the run that worked? I'm assuming that the concretize log attached above is from the run that did not work.

Also, what does `brew list` return, as well as `brew info pkg-config`?

Thanks!